### ISSUE TITLE
Health of VS Code Kubernetes Tools

### ISSUE THREAD

**Issue by caniszczyk** [cncf/toc#1241]
The TOC determined that the VS Code Kubernetes Tools project's health is currently unknown and isn't fully onboarded. This issue is created to track the Roadmap request to bring the project back into alignment with expectations of sandbox projects.

https://github.com/cncf/sandbox/issues/181

Repo: https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools

Maintainers: https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/blob/master/OWNERS.md

**Comment by caniszczyk** [comment]
I will reach out to the project and get a response from them, let's give them a couple weeks before we make any votes/decisions as this seems to be popular...

<img width="815" alt="Screenshot 2024-01-16 at 10 26 40 AM" src="https://github.com/cncf/toc/assets/63777/347796ce-d71d-4432-8442-33d690ed8771">

**Comment by TheFoxAtWork** [comment]
Thanks @caniszczyk . Let's try to get a response by March 1st 2024.  CC @bnookala , @lstocchi , @itowlson

**Comment by itowlson** [comment]
Kia ora @TheFoxAtWork - sorry, I'm no longer involved with this project.  cc @Tatsinnit who I believe is working on the extension.

**Comment by squillace** [comment]
yup, I'm also here along with Tats. I had thought we'd completed the onboarding process, but I can help finish that up. This extension is widely used as a dependency for other K8s-related extensions.

**Comment by TheFoxAtWork** [comment]
@squillace and @tatsinnit, thanks for the update. [Please work with CNCF staff to resolve outstanding onboarding items](https://github.com/cncf/toc/issues/745#issuecomment-2192484425). @krook when onboarding is complete can you follow up here so we may close this issue?

**Comment by krook** [comment]
> @krook when onboarding is complete can you follow up here so we may close this issue?
Will do!

**Comment by angellk** [comment]
@krook is onboarding complete?

**Comment by krook** [comment]
> @krook is onboarding complete?

Not yet, there are a few tasks outstanding, two of which staff has been working on this week (Insights onboarding and closing on the trademark transfer). For the others, I'll bump the onboarding issue.

**Comment by krook** [comment]
An update, @squillace has met with the development team to understand the remaining open items on the onboarding checklist. We will sit down at KubeCon to work through and close out the final items here on site.

**Comment by linsun** [comment]
Tagging all maintainers of the project to reply @Tatsinnit @ReinierCC @tejhan @hsubramanianaks @qpetraroia @squillace **within a month**:

The TOC has identified this project in need of a health/activity check up, **we're requesting your assistance in answering the following questions:**
    * Are you familiar with this project?
    * Do you know if the project is active, perhaps outside of github?
    * Do you know if it is maintained and if so who is maintaining it?
    * Are you using or do you know someone who is using the project?

We keep the **health response window open for one-month**, to allow projects to provide the TOC with context around their health or inactivity before the TOC makes a determination to vote for archive or to provide a roadmap back to health for the project.

**Comment by Tatsinnit** [comment]
> Tagging all maintainers of the project to reply [@Tatsinnit](https://github.com/Tatsinnit) [@ReinierCC](https://github.com/ReinierCC) [@tejhan](https://github.com/tejhan) [@hsubramanianaks](https://github.com/hsubramanianaks) [@qpetraroia](https://github.com/qpetraroia) [@squillace](https://github.com/squillace) **within a month**:
> 
> The TOC has identified this project in need of a health/activity check up, **we're requesting your assistance in answering the following questions:** * Are you familiar with this project? * Do you know if the project is active, perhaps outside of github? * Do you know if it is maintained and if so who is maintaining it? * Are you using or do you know someone who is using the project?
> 
> We keep the **health response window open for one-month**, to allow projects to provide the TOC with context around their health or inactivity before the TOC makes a determination to vote for archive or to provide a roadmap back to health for the project.

Hiya @linsun ,  ❤ Thank you for reaching out and for your efforts in keeping track of project health. Please find our responses below:  

- **Are you familiar with this project?**  
  Yes, all of us listed in the updated [`OWNERS.md`](https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/blob/master/OWNERS.md) are familiar with this project.

- **Do you know if the project is active, perhaps outside of GitHub?**  
  The project is actively used as a tool but GitHub is the only place this code is maintained. Further to add this project has been in a relatively stagnant mode. However, we have Dependabot enabled to monitor dependencies, and we continue to track changes. Recently, we’ve had a few PRs merged and have encouraged new contributors, which is helping bring steady activity back.  

- **Do you know if it is maintained, and if so, who is maintaining it?**  
  Yes, the project is maintained. The latest updates on maintainers can be found in the (`OWNERS.md`)[https://github.com/vscode-kubernetes-tools/vscode-kubernetes-tools/blob/master/OWNERS.md] file.  

- **Are you using or do you know someone who is using the project?**  
  Yes, I know a few folks who rely on this extension as part of their daily workflow.  

Thanks again for checking in!  **As always a gentle, humble fyi for any addition and correction of the above mention to kind fellow crew** + @ReinierCC + @tejhan +  @hsubramanianaks + @qpetraroia and @squillace 🙏

**Comment by krook** [comment]
> [@krook](https://github.com/krook) is onboarding complete?

Yes, it's now complete. Thank you @Tatsinnit and @squillace.

**Comment by angellk** [comment]
Now that onboarding is complete - closing this issue. Thanks @Tatsinnit and @squillace for completing onboarding and ensuring the project maintains the standards to remain in CNCF Sandbox.
